src/components/GoalTableList/GoalTableList.tsx

```diff
@@ -121,38 +129,36 @@ export const GoalTableList = <T extends Partial<NonNullable<GoalByIdReturnType>>
                         width: 90,
                     },
                     {
-                        content: nullable(goal._achivedCriteriaWeight, (weight) => (
-                            <CircleProgressBar value={weight} />
-                        )),
+                        content: goal._achivedCriteriaWeight != null && goal.id != null && (
```

Comment:
why not keep nullable?

Reply:
'cause `nullable` doesn't work as typeguard and by usage doesn't exclude `null` value from variable